src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -121,6 +122,12 @@
         public double PCT_90X;
         /** The fraction of bases that attained at least 100X sequence coverage in post-filtering bases. */
         public double PCT_100X;
+
+        /** The theoretical HET SNP sensitivity. */
+        public double HET_SNP_SENS;
+
+        /** The Q Score of the theoretical HET SNP sensitivity. */
```

Comment:
Indicate that this is a phred-scaled Q score (unless 'Q' score in Picard always mean phred-scaled, in which case disregard).